### PR TITLE
dts: Add Samsung Galaxy J5 2015 (SM-J500H) (Rev. 5)

### PR DESCRIPTION
**Pull request by nergzd723**
Add support for MSM8216 Galaxy J5 (2015) 3G/

### CODE REVIEW

File: dts/msm8216-samsung-r05.dts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <206 0>;
```
Comment (Minecrell) [inline dts/msm8216-samsung-r05.dts:9]: Can you try if

```suggestion
	qcom,msm-id = <248 0>;
```

works? That should be more correct. Thanks.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ and then loaded by lk2nd.
 - Samsung Galaxy A3 (2015) - SM-A300FU
 - Samsung Galaxy A5 (2015) - SM-A500F, SM-A500FU
 - Samsung Galaxy J5 (2015) - SM-J500FN
+- Samsung Galaxy J5 (2015) - SM-J500H
```
Comment (Minecrell) [inline README.md:26]: Please put it together on the line above (separated by comma)

File: dts/msm8216-samsung-r05.dts (hunk 1 above)
Comment (nergzd723) [reply dts/msm8216-samsung-r05.dts]: Yes, it works.

File: README.md (hunk 2 above)
Comment (nergzd723) [reply README.md]: Done.

**Comment by nergzd723** [comment]
I've updated my fork for merge.

File: README.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ and then loaded by lk2nd.
 ## Supported SoCs
 - MSM8916
 - MSM8929
+- MSM8216
 - MSM8939
```
Comment (Minecrell) [inline README.md:19]: I think we should keep this list alphabetically ordered.

File: dts/rules.mk [hunk 4]
```diff
@@ -15,5 +15,6 @@ DTBS += \
 	$(LOCAL_DIR)/msm8916-samsung-r08.dtb \
 	$(LOCAL_DIR)/msm8929-samsung-r00.dtb \
 	$(LOCAL_DIR)/msm8929-samsung-r04.dtb \
+	$(LOCAL_DIR)/msm8216-samsung-r05.dtb \
 	$(LOCAL_DIR)/msm8939-mtp.dtb
```
Comment (Minecrell) [inline dts/rules.mk:19]: And this one too. (it should be after `apq8016`)

File: README.md (hunk 3 above)
Comment (nergzd723) [reply README.md:19]: Done

File: dts/msm8216-samsung-r05.dts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <248 0>;
+	qcom,board-id = <0xF808FF01 5>;
+
+	j53g-eur {
+		model = "Samsung Galaxy J5 2015 (SM-J500H)";
+		compatible = "samsung,j53gxx", "qcom,msm8916", "lk2nd,device";
```
Comment (Minecrell) [inline dts/msm8216-samsung-r05.dts:14]: I think we should either do

```suggestion
		compatible = "samsung,j53g", "qcom,msm8916", "lk2nd,device";
```

here or

```suggestion
		compatible = "samsung,j53g-eur", "qcom,msm8916", "lk2nd,device";
```

The `xx` is more of an Android convention.

Comment (nergzd723) [reply dts/msm8216-samsung-r05.dts:14]: I can agree

Comment (nergzd723) [reply dts/msm8216-samsung-r05.dts]: fixed

**Comment by Minecrell** [comment]
I removed the trailing spaces you added on one of the lines and pushed it to master. Thank you!

**Pull request closed without merging**